app/src/js/AstroMath.js

```diff
@@ -112,7 +113,9 @@ export default class AstroMath {
     if (normalRange) {
       convertedLng *= -1;
     } else {
-      convertedLng = Math.abs(convertedLng - 360);
+      if (convertedLng < 0){
```

Comment:
In Trent's gitter message, for the lon 360 domain and lon west direction, he subtracts the angle from 360. See:
```elif ((combobox180.value == "360") and (comboboxEW.value == "West")):
            if (X < 0):
                X = 360 + X
            X = 360 - X
```
Do we need to include this in the domainToPositiveWest function?

Reply:
We should also remove the code:  
``
if (projection === "EPSG:4326") {
       convertedLon -= 180;
 }
```